application/common/components/MfaBackendWebAuthn.php

```diff
@@ -158,6 +158,9 @@ public function verify(int $mfaId, $value, string $rpOrigin = '', string $verify
             if ($webauthnCount < 1) {
                 throw new NotFoundHttpException("No MFA Webauthn record found for MFA ID: " . $mfa->id, 1659637860);
             }
+
+            $mfa->setLastUsed();
```

Comment:
It looks like this is recording a last-used date before verifying that the response was valid. Is that correct?

Reply:
Yes, it does look like I got that wrong. Thanks for spotting it.

Reply:
What's worse, as I got to looking at it further, I'm not sure it's going to work at all. I tried to call `setLastUsed` on an `Mfa` object, but I defined the function on `MfaWebauthn`. It appears that the validation is delegated to the MFA API, which doesn't give any indication as to which webauthn instance passes validation.